{"sections":[],"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error"},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"text":"error","kind":"identifier"},{"kind":"text","text":": "},{"text":"APIError","kind":"typeIdentifier","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","preciseIdentifier":"s:6OpenAI8APIErrorV"}]}]}],"variants":[{"paths":["\/documentation\/openai\/apierrorresponse\/error"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"modules":[{"name":"OpenAI"}],"title":"error","externalID":"s:6OpenAI16APIErrorResponseV5errorAA0C0Vvp","role":"symbol","roleHeading":"Instance Property","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"error","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:6OpenAI8APIErrorV","text":"APIError","kind":"typeIdentifier"}],"symbolKind":"property"},"references":{"doc://OpenAI/documentation/OpenAI/APIErrorResponse/error":{"fragments":[{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"text":"error","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"APIError","preciseIdentifier":"s:6OpenAI8APIErrorV"}],"title":"error","kind":"symbol","url":"\/documentation\/openai\/apierrorresponse\/error","role":"symbol","type":"topic","abstract":[],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/error"},"doc://OpenAI/documentation/OpenAI/APIError":{"fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"kind":"identifier","text":"APIError"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","title":"APIError","type":"topic","abstract":[],"url":"\/documentation\/openai\/apierror","navigatorTitle":[{"kind":"identifier","text":"APIError"}]},"doc://OpenAI/documentation/OpenAI":{"title":"OpenAI","type":"topic","url":"\/documentation\/openai","kind":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","role":"collection","abstract":[]},"doc://OpenAI/documentation/OpenAI/APIErrorResponse":{"abstract":[],"role":"symbol","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse","navigatorTitle":[{"kind":"identifier","text":"APIErrorResponse"}],"title":"APIErrorResponse","kind":"symbol","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"APIErrorResponse","kind":"identifier"}],"type":"topic","url":"\/documentation\/openai\/apierrorresponse"}}}